selenium_tests/cross_listing/cross_listing_base_test_case.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from urlparse import urljoin
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_common.pin.page_objects.pin_login_page_object \
+    import PinLoginPageObject
+
+from selenium_tests.account_admin.page_objects.account_admin_dashboard_page_object \
+    import AccountAdminDashboardPage
+from selenium_tests.cross_listing.page_objects.cross_listing_index_page \
+    import IndexPagePageObject
+
+
+class CrossListingBaseTestCase(BaseSeleniumTestCase):
```

Comment:
Update: Sweet!  Was able to figure it out and get it working to make it more DRYer --- will put in a PR for this or the change will be bundled with TLT-1589.